### PR TITLE
20171216 readme.md

### PR DESCRIPTION
**Pull request by AndyHee**
Major rewrite and adding of numerous screenshots.

https://social.diekershoff.de/display/f3ad7b1c795a2fa91286223624551418

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,28 @@
...
+
+Join today and [get your Friendica profile!](https://dir.friendica.social/servers 'Join Friendica today!')
+
 
-Friendica is a communications platform for integrated social communications utilising decentralised communications and linkage to several indie social projects - as well as popular mainstream providers.
 
-Our mission is to free our friends and families from the clutches of data-harvesting corporations, and pave the way to a future where social communications are free and open and flow between alternate providers as easily as email does today.
+### Friendica Screenshots
 
+![Alt text](/images/screenshots/friendica-frio-mobile-profle-1.png?raw=true "Frio theme on a mobile")
```
Comment (MrPetovan) [inline README.md]: Shouldn't `Alt text` be replaced by actual text alternative?

Comment (tobiasd) [reply README.md]: It should, yes.

**Comment by AndyHee** [comment]
You are right, good man.

I just figured out how to do captions.

I'll make a bit more descriptive specifying what is shown, i.e. timeline, composer window.

Sorry bear with me. It should be ready shortly.

**Comment by AndyHee** [comment]
Done. Please see changes.

**Comment by AndyHee** [comment]
Ahh... now I understand. I just opened it in lynx and saw the problem. I confused the popup over the image with the Alt-Text. Thank you for you patience.

**Comment by MrPetovan** [comment]
No, thank you for your time!

**Comment by tobiasd** [comment]
Thanks for the update!

**Pull request merged**